app/src/main/res/values-pt/strings.xml

```diff
@@ -116,9 +117,9 @@
     <string name="alternative_highlight">Destaque alternativo</string>
     <string name="dlg_delete_all_favorites">Remover todos os favoritos?</string>
     <string name="dlg_delete_all_favorites_delete_all">Remover tudo</string>
-
+    
```

Comment:
Please don't introduce trailing spaces. The IDE should remove them automatically when you autoformat the file. In general I like to have pure formatting changes in a separate commit so that code changes are not hidden.